android/lib/models/art_item_model.dart

```diff
@@ -1,15 +1,77 @@
 import 'package:android/models/models.dart';
-import 'package:android/models/user_model.dart';
+import 'package:android/data/data.dart';
 
 class ArtItem {
-  final String name;
-  final String description;
-  final String imageUrl;
-  final User artist;
-
-  ArtItem(
-      {required this.name,
-      required this.description,
-      required this.imageUrl,
-      required this.artist});
+  final int id;
+  final ArtItemInfo artItemInfo;
+  final User creator;
+  final DateTime creationDate;
+  final User owner;
+  final bool onAuction;
+  final String? auction;
+  final double? lastPrice;
+  final List<String> commentList;
+  final List<User> bookmarkedBy;
+
+  ArtItem({
+    required this.id,
+    required this.artItemInfo,
+    required this.creator,
+    required this.creationDate,
+    required this.owner,
+    required this.onAuction,
+    this.auction,
+    this.lastPrice,
+    required this.commentList,
+    required this.bookmarkedBy,
+  });
+
+  factory ArtItem.fromJson(Map<String, dynamic> json) {
+    return ArtItem(
+      id: json['id'],
+
+      artItemInfo: json['artItemInfo'] != null
```

Comment:
In order to escape from null type checks, following this approach is not the correct way in my opinion.